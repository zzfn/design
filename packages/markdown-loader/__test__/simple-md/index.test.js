const fs = require('fs');
const path = require('path');
const prettier = require('prettier');
const loader = require('../../lib/index').default;

const md = fs.readFileSync(path.resolve(__dirname, 'demo.md'), 'utf8');

it('simple markdown', () => {
  const code = prettier.format(loader.bind({})(md), { parser: 'babel' });
  expect(code).toMatchSnapshot();
});
