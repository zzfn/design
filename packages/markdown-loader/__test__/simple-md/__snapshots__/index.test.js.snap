// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple markdown 1`] = `
"import React, { useState } from \\"react\\";
export default function (props) {
  const [isUsage, setIsUsage] = useState(false);
  const [showChangelog, setShowChangelog] = useState(false);
  const lang = localStorage.getItem(\\"arco-lang\\") || \\"zh-CN\\";
  return (
    <span style={props.style} className=\\"markdown-body\\">
      <h1 id=\\"title\\">Title</h1>
      <p>Description.</p>
    </span>
  );
}
"
`;
